lua/telescope/actions.lua

```diff
@@ -83,6 +83,22 @@ function actions.goto_file_selection(prompt_bufnr)
   end
 end
 
+function actions.goto_file_selection_edit(prompt_bufnr)
+	goto_file_selection(prompt_bufnr, "e")
```

Comment:
can you change this to two space indenting?

Reply:
Sry my mistake. Fixed it